### ISSUE TITLE
Pretrained batch normalization model

### ISSUE THREAD

**Issue by niufuren** [lim0606/caffe-googlenet-bn-experiment#2]
@lim0606 is it possible to share your pretrained googlenet model? I'd like to experiment a fine-tuning with it.
